### PR TITLE
Add test cases for java.lang.Integer and java.lang.Float

### PR DESCRIPTION
**Pull request by loint**

### CODE REVIEW

File: java/lang/Integer/IntegerTest.cpp [hunk 1]
```diff
@@ -241,3 +241,19 @@ TEST (JavaLang, IntegerStringValue) {
...
 }
+
+TEST (JavaLang, IntegerShortValue) {
+    // Input different values of type int to compare to the result of Integer::shortValue
+    Integer positiveInteger = 1302;
+    Integer negativeInteger = -1302;
+
+    // Make a comparison between expected result and the real result
+    int expectedResult = 1302;
+    int realResult = positiveInteger.shortValue();
```
Comment (dthongvl) [inline java/lang/Integer/IntegerTest.cpp:252]: This variable should be **short** data type

File: java/lang/Float/FloatTest.cpp [hunk 2]
```diff
@@ -104,4 +104,21 @@ TEST (JavaLang, FloatParseFloat) {
 	Float validFloatPointer = Float::parseFloat("6");
 	float expect = 6;
 	ASSERT_EQUAL(expect, validFloatPointer.floatValue());
+}
+
+
+TEST (JavaLang, FloatToString) {
+	// Input different values of t-
```
Comment (dthongvl) [inline java/lang/Float/FloatTest.cpp:111]: Should fix this comment style

File: java/lang/Integer/IntegerTest.cpp [hunk 3]
```diff
@@ -129,3 +129,42 @@ TEST (JavaLang, IntegerCompareTo) {
...
+TEST (JavaLang, IntegerToString) {
+    // Input different values of type int to compare to the result of Integer::toString
+    Integer zeroInteger = 0;
+    Integer oneInteger = 1;
+    Integer positiveInteger = 10;
+    Integer maxInteger = 2147483647;
+    Integer minInteger = -2147483647;
+
+    // Make a comparison between expected result and the real result
+    char *expect = "0";
```
Comment (dthongvl) [inline java/lang/Integer/IntegerTest.cpp:142]: Should use **string** instead of char *
We defined **string** = char *

**Review by neacao — APPROVED** [review]

**Review by dthongvl — APPROVED** [review]
Approved

**Pull request merged**